src/adapter/src/coord.rs

```diff
@@ -501,6 +502,9 @@ pub struct Coordinator {
     /// active connections.
     active_conns: BTreeMap<ConnectionId, ConnMeta>,
 
+    /// Number of active dataflows per cluster
+    active_dataflows_per_cluster: BTreeMap<ClusterId, usize>,
```

Comment:
I'm worried that this will be hard to keep consistent. Is it possible to derive this from `pending_peeks` and `active_subscribes` instead of having a separate field?